src/datasets/download/download_manager.py

```diff
@@ -377,7 +377,7 @@ def iter_files(self, paths: Union[str, List[str]]):
         """
         return FilesIterable.from_paths(paths)
 
-    def extract(self, path_or_paths, num_proc=None):
+    def extract(self, path_or_paths):
```

Comment:
"Deprecating a param" does not mean we can remove the parameter abruptly. This is what we should do instead:
```python
def extract(self, path_or_paths, num_proc="deprecated"):
    """Extract given path(s).

    Args:
        path_or_paths: path or `list`/`dict` of path of file to extract. Each
            path is a `str`.
        num_proc: Use multi-processing if `num_proc` > 1 and the length of
            `path_or_paths` is larger than `num_proc`

            <Deprecated version="2.6.2">

            Pass `DownloadConfig(num_proc=<num_proc>)` to the initializer instead.

            </Deprecated>

    Returns:
        extracted_path(s): `str`, The extracted paths matching the given input
            path_or_paths.

    Example:

    ```py
    >>> downloaded_files = dl_manager.download('https://storage.googleapis.com/seldon-datasets/sentence_polarity_v1/rt-polaritydata.tar.gz')
    >>> extracted_files = dl_manager.extract(downloaded_files)
    ```
    """
    if num_proc != "deprecated":
        warnings.warn(
            "'num_proc' was deprecated in version 2.6.2 and will be removed in 3.0.0. Pass `DownloadConfig(num_proc=<num_proc>)` to the initializer instead.",
            FutureWarning,
        )
    # the rest of the code ...
```

Reply:
Thanks for your help. I commited the changes as you said.